tests/foreman/sys/test_dynflow.py

```diff
@@ -0,0 +1,41 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Dynflow
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Dynflow
+
+:Requirement: Dynflow
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo import ssh
+from robottelo.decorators import tier2
+from robottelo.test import TestCase
+
+
+class DynflowTestCase(TestCase):
+    """Dynflow tests"""
+
+    @tier2
+    def test_positive_setup_dynflow(self):
+        """Set dynflow parameters, restart it and check it adheres to them
+
+        :id: a5aaab5e-bc18-453e-a284-64aef752ec88
+
+        :expectedresults: Correct dynflow processes are running, respecting settings
+
+        :CaseImportance: High
+        """
+        ssh.command(
+            "cd /etc/foreman/dynflow/ && cp worker-hosts-queue.yml test.yml && sed -i s/5/6/"
+            "test.yml && systemctl restart 'dynflow-sidekiq@test' && res=1 && while"
+            "[ $res -eq 1 ]; do systemctl status 'dynflow-sidekiq@test' -l | "
+            "grep ' of 6 busy' && res=$(echo $?); done"
```

Comment:
This kind of loop can be simplified to:
```
while ! systemctl status 'dynflow-sidekiq@test' -l | grep -q ' of 6 busy' ; do sleep 0.5 ; done
```

Shouldn't you clean up after yourself, i.e. remove test.yml file?

Could we put individual commands into list and just join them on '&&' when calling? I think it will be little easier to read. Something like:
```python
commands = [
    "cd /etc/foreman/dynflow/",
    "cp worker-hosts-queue.yml test.yml"
]
ssh.command(" && ".join(commands))
```

Reply:
Good points, I'll do that

Reply:
Done